ewjuviecmd.py

```diff
@@ -178,20 +245,8 @@
 
 			user_data.persist()
 
-			# Tell the player their slime level increased and/or a poudrin was found.
-			if was_levelup or poudrin:
-				response = ""
-
-				if poudrin:
-					if poudrinamount == 1:
-						response += "You unearthed a slime poudrin! "
-					elif poudrinamount == 2:
-						response += "You unearthed two slime poudrins! "
-
-					ewstats.change_stat(user = user_data, metric = ewcfg.stat_lifetime_poudrins, n = poudrinamount)
-
-					ewutils.logMsg('{} has found {} poudrin(s)!'.format(cmd.message.author.display_name, poudrinamount))
-
+			# Tell the player their slime level increased and/or they unearthed an item.
+			if was_levelup:
 				if was_levelup:
```

Comment:
this doesn't seem right. it checks for level up twice and won't send a message if an item was found, but no level up occurred